parsec/core/fs/storage/chunk_storage.py

```diff
@@ -96,6 +96,44 @@ async def is_chunk(self, chunk_id: ChunkID) -> bool:
             manifest_row = cursor.fetchone()
         return bool(manifest_row)
 
+    async def get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+
+        local_chunk_ids = []
+        bytes_id_list = [(id.bytes,) for id in chunk_id]
+
+        async with self._open_cursor() as cursor:
+            # Can't use execute many with SELECT so we have to make a temporary table filled with the needed chunk_id
+            # and intersect it with the normal table
+            # create random name for the temporary table to avoid asynchronous errors
+            table_name = "temp" + secrets.token_hex(12)
+            table_name = "".join(chr for chr in table_name if chr.isalnum())
```

Comment:
Isn't `table_name` already alphanum only? If it acts as a safety net then an `assert` would be a better fit.